File: Samples for Parallel Programming with the .NET Framework/[C#][VB][C++][F#]-Samples for Parallel Programming with the .NET Framework/C# and C++ and F# and VB/C#,C++,F#,VB/Raytracer/Raytracer_VisualBasic/Light.vb
'--------------------------------------------------------------------------
' 
'  Copyright (c) Microsoft Corporation.  All rights reserved. 
' 
'  File: Light.vb
'
'--------------------------------------------------------------------------

Public Class Light
    Public Position As Vector
    Public Color As Color

    Public Sub New(ByVal position As Vector, ByVal color As Color)
        Me.Position = position
        Me.Color = color
    End Sub
End Class

File: Samples for Parallel Programming with the .NET Framework/[C#][VB][C++][F#]-Samples for Parallel Programming with the .NET Framework/C# and C++ and F# and VB/C#,C++,F#,VB/Raytracer/Raytracer_CSharp/Light.cs
//--------------------------------------------------------------------------
// 
//  Copyright (c) Microsoft Corporation.  All rights reserved. 
// 
//  File: Light.cs
//
//--------------------------------------------------------------------------

namespace Microsoft.ParallelComputingPlatform.ParallelExtensions.Samples
{
    class Light
    {
        public Vector Pos;
        public Color Color;

        public Light(Vector pos, Color color) { Pos = pos; Color = color; }
    }
}
